src/DSharp.Script/src/System/Collections/Interfaces.js

```diff
@@ -0,0 +1,16 @@
+// Contracts
+
+function ICollection() { }
+function ICollection_$1() { }
+function IReadOnlyCollection_$1() { }
+
+function Dictionary_$2() { }
```

Comment:
Move to a new script file called Dictionary.js, under System/Generic/Collections folder